redfish-core/lib/log_services.hpp

```diff
@@ -2741,6 +2835,12 @@ inline void requestRoutesDBusEventLogEntry(App& app)
                     "/redfish/v1/Systems/system/LogServices/EventLog/Entries/" +
                     std::to_string(*id) + "/attachment";
```

Comment:
```
boost::urls::format()
```

Reply:
hmm, this function shouldn't have been new. need to investigate.

Reply:
yea, not sure why this is showing as a new function.  it's right here - https://github.com/ibm-openbmc/bmcweb/blob/1110/redfish-core/lib/log_services.hpp#L2305.  also going through the commits one at a time won't show it.